camel/sync/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,4 +35,9 @@
     <routeContextRef ref="drupalAuthentication"/>
   </camelContext>
 
+  <!-- configuration of activemq component -->
+  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="islandora.brokerUrl" value="${islandora.brokerUrl}"/>
```

Comment:
@DiegoPino sorry I totally missed this.

Because you could have ```stomp://localhost:61613``` as your brokerURL, so I would not hardcode any part of it.